lips/lip-1.mdx

```diff
@@ -492,84 +458,235 @@ Represents a national ID.
+The receiver VASP examines the sender KYC information and is either not satisfied the payment can proceed, needs more time to process the KYC information, or requires additional information to determine if the payment can proceed. It includes a command to abort the payment with an appropriate error code.
 
-# Recipient Signature
-Once the recipient side is comfortable that it has received appropriate information and is ready for the transaction to go on-chain, the recipient side will provide their signature in order to support dual attestation. The receiver VASP will sign with the receiver compliance private key. The following is a concrete example of how to generate the Travel Rule dual attestation signable and the recipient signature using Libra Python Client SDK:
+* `Receiver.Status.Status`: `abort`
+* `Receiver.Status.abort_code`: one of `no-kyc-needed` or `rejected`.
+
+The sender can initiate a payment on-chain in case of an abort with `no-kyc-needed`.
+
+### RSEND -> READY (Step 7)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The Sender VASP examines the KYC information from the Receiver and is satisfied the payment can proceed.
+
+* `Sender.Status.Status`: `ready_for_settlement`
+
+The payment should be executed on-chain by the sender (or settled in any other way) following the success of the command.
+
+### RSEND -> SABORT (Step 9)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The sender VASP receives the Receiver KYC information and decides it cannot proceed with the payment. It issues an abort command:
+
+* `Sender.Status.Status`: `abort`
+* `Sender.Status.abort_code`: `rejected`.
+
+## Steps of the protocol: The soft-match states and flows
+
+A _soft-match_ occurs when a VASP checks provided KYC, but cannot disambiguate whether a party to the payment is potentially high-risk or sanctioned, just with the information provided. This may occur because the identifiers used (names, dates of birth, addresses) may partially match and are noisy. In such cases the VASP may require further information from the other VASP to process the payment, some of which may require manual intervention and interaction.
+
+The flows below allow the Receiver VASP and the Sender VASP to request additional KYC information from each other.
+
+### Receiver soft-match (SINIT -> RSOFT -> SSOFTSEND)
```

Comment:
Do we write anywhere how long a VASP should expect to wait for a soft match response?

Reply:
Updated to include DIP 5 link.  Updated to use wording on request sender address header

Reply:
Updated to say that soft matches are suggested to be cleared within 24 hours